pkg/commands/git.go

```diff
@@ -261,8 +261,13 @@ func (c *GitCommand) RenameCommit(name string) error {
 }
 
 // Fetch fetch git repo
-func (c *GitCommand) Fetch() error {
-	return c.OSCommand.RunCommand("git fetch")
+func (c *GitCommand) Fetch(unamePassQuestion func(string) string, canSskForCredentials bool) error {
+	return c.OSCommand.DetectUnamePass("git fetch", func(question string) string {
+		if canSskForCredentials {
+			return unamePassQuestion(question)
+		}
+		return "-"
```

Comment:
why are we returning a `-` here?

Reply:
The live execute function only writes to the stdin if the return string is longer than 1 so i need to return something otherwise it would hang